src/routes/view-content/bible-menu/BookChapterSelectorPane.svelte

```diff
@@ -254,10 +270,10 @@
                 <span class="ms-4">{currentStep.backText}</span>
             </button>
         {:else}
-            <h3 class="my-2 font-bold">{currentStep.title}</h3>
+            <h3 class="my-2 block font-bold">{currentStep.title}</h3>
```

Comment:
From here on down are some subtle CSS changes that make things feel a bit smoother. I was having issues when selecting verses on books with lots of chapters. Weird scroll issues, but those are fixed now.